presto-spi/src/main/java/com/facebook/presto/spi/security/AccessControlContext.java

```diff
@@ -32,8 +32,19 @@ public class AccessControlContext
     private final WarningCollector warningCollector;
     private final RuntimeStats runtimeStats;
     private final Optional<QueryType> queryType;
+    private final Optional<String> catalogName;
+    private final Optional<String> sqlNamespace;
```

Comment:
call this schema for consistency with the terms we use throughout presto (and update other usages similarly)
```suggestion
    private final Optional<String> schema;
```